fcrepo-persistence-ocfl/src/main/java/org/fcrepo/persistence/ocfl/impl/CreateUpdatePersister.java

```diff
@@ -43,30 +44,54 @@
  * @author dbernstein
  * @since 6.0.0
  */
-public class RDFSourcePersister extends AbstractPersister {
+public class CreateUpdatePersister extends AbstractPersister {
```

Comment:
While I don't think its doing much harm, I'm not sure i see a strong reason to merge RDF and Non-RDF persisters together here. Its a short implementation, but it seems like the core action taken is different based on rdf vs non-rdf. I guess I'm not going to push too hard for reverting the merge, but it might be a little less in the "single responsibility" department.